src/test/components/EmptyThreadIndicator.test.js

```diff
@@ -3,8 +3,6 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // @flow
-jest.mock('react-dom');
-
```

Comment:
The automatic cleanup from the testing library was failing badly because of that. TBH I don't know why this was here, the test works just fine without it, and it's very curious that it was working at all before...

Reply:
Oof. Yeah I have no recollection.